rpm/tarantool-metrics.spec

```diff
@@ -57,6 +57,9 @@ cp -rv cartridge %{br_luapkgdir}
      %{luapkgdir}/metrics/default_metrics/tarantool/slab.lua
      %{luapkgdir}/metrics/default_metrics/tarantool/spaces.lua
      %{luapkgdir}/metrics/default_metrics/tarantool/system.lua
+%dir %{luapkgdir}/metrics/default_metrics/cartridge
```

Comment:
we should rename files:

metrics/cartridge.lua for module an entry point 
metrics/cartridge/issues.lua for module files